<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ItemShapeWidget</class>
 <widget class="QWidget" name="ItemShapeWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>681</width>
    <height>60</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>620</width>
    <height>60</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Meiryo UI</family>
    <pointsize>12</pointsize>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="sizeConstraint">
    <enum>QLayout::SetMinimumSize</enum>
   </property>
   <property name="leftMargin">
    <number>20</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="frame_2">
     <property name="minimumSize">
      <size>
       <width>88</width>
       <height>0</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>14</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>4</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="posX">
        <property name="maximumSize">
         <size>
          <width>12</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Yu Gothic</family>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">#posX
{
color: rgb(102, 102, 102);
}
#posX:disabled
{
color: rgb(140, 140, 140);
}</string>
        </property>
        <property name="text">
         <string>X: </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="posXEdit">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Yu Gothic</family>
          <pointsize>10</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="focusPolicy">
         <enum>Qt::ClickFocus</enum>
        </property>
        <property name="toolTip">
         <string>Position X</string>
        </property>
        <property name="styleSheet">
         <string notr="true">QDoubleSpinBox
{
background-color: transparent;
border: 1px solid transparent;
border-bottom-color:#ccc
}
QDoubleSpinBox::up-arrow
{
image: url(:/uiIcon/src/ui/resource/icon/up_arrow.png);
width:7px; height:7px;

}
QDoubleSpinBox::down-arrow
{
image: url(:/uiIcon/src/ui/resource/icon/down_arrow.png);
}
QDoubleSpinBox::down-arrow:disabled
{
image: url(:/uiIcon/src/ui/resource/icon/down_arrow_disabled.png);
}
QDoubleSpinBox::up-arrow:disabled
{
image: url(:/uiIcon/src/ui/resource/icon/up_arrow_disabled.png);
}</string>
        </property>
        <property name="wrapping">
         <bool>false</bool>
        </property>
        <property name="frame">
         <bool>false</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="readOnly">
         <bool>false</bool>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::UpDownArrows</enum>
        </property>
        <property name="accelerated">
         <bool>false</bool>
        </property>
        <property name="showGroupSeparator" stdset="0">
         <bool>false</bool>
        </property>
        <property name="prefix">
         <string/>
        </property>
        <property name="minimum">
         <double>-200.000000000000000</double>
        </property>
        <property name="maximum">
         <double>1000.000000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame_3">
     <property name="minimumSize">
      <size>
       <width>88</width>
       <height>0</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>4</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>12</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="posY">
        <property name="minimumSize">
         <size>
          <width>12</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>12</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Yu Gothic</family>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">#posY
{
color: rgb(102, 102, 102);
}
#posY:disabled
{
color: rgb(140, 140, 140);
}</string>
        </property>
        <property name="text">
         <string>Y: </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="posYEdit">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Yu Gothic</family>
          <pointsize>10</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="focusPolicy">
         <enum>Qt::ClickFocus</enum>
        </property>
        <property name="toolTip">
         <string>Position Y</string>
        </property>
        <property name="styleSheet">
         <string notr="true">QDoubleSpinBox
{
background-color: transparent;
border: 1px solid transparent;
border-bottom-color:#ccc
}
QDoubleSpinBox::up-arrow
{
image: url(:/uiIcon/src/ui/resource/icon/up_arrow.png);
width:7px; height:7px;

}
QDoubleSpinBox::down-arrow
{
image: url(:/uiIcon/src/ui/resource/icon/down_arrow.png);
}
QDoubleSpinBox::down-arrow:disabled
{
image: url(:/uiIcon/src/ui/resource/icon/down_arrow_disabled.png);
}
QDoubleSpinBox::up-arrow:disabled
{
image: url(:/uiIcon/src/ui/resource/icon/up_arrow_disabled.png);
}</string>
        </property>
        <property name="frame">
         <bool>false</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="prefix">
         <string/>
        </property>
        <property name="minimum">
         <double>-200.000000000000000</double>
        </property>
        <property name="maximum">
         <double>1000.000000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>92</width>
       <height>0</height>
      </size>
     </property>
     <property name="cursor">
      <cursorShape>SizeVerCursor</cursorShape>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>12</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>4</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="width">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>16</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Yu Gothic</family>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">#width
{
color: rgb(102, 102, 102);
}
#width:disabled
{
color: rgb(140, 140, 140);
}</string>
        </property>
        <property name="text">
         <string>width: </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="widthEdit">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Yu Gothic</family>
          <pointsize>10</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="focusPolicy">
         <enum>Qt::StrongFocus</enum>
        </property>
        <property name="toolTip">
         <string>Image Width</string>
        </property>
        <property name="styleSheet">
         <string notr="true">QDoubleSpinBox
{
background-color: transparent;
border: 1px solid transparent;
border-bottom-color:#ccc
}
QDoubleSpinBox::up-arrow
{
image: url(:/uiIcon/src/ui/resource/icon/up_arrow.png);
width:7px; height:7px;

}
QDoubleSpinBox::down-arrow
{
image: url(:/uiIcon/src/ui/resource/icon/down_arrow.png);
}
QDoubleSpinBox::down-arrow:disabled
{
image: url(:/uiIcon/src/ui/resource/icon/down_arrow_disabled.png);
}
QDoubleSpinBox::up-arrow:disabled
{
image: url(:/uiIcon/src/ui/resource/icon/up_arrow_disabled.png);
}</string>
        </property>
        <property name="frame">
         <bool>false</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="prefix">
         <string/>
        </property>
        <property name="minimum">
         <double>0.000000000000000</double>
        </property>
        <property name="maximum">
         <double>1050.000000000000000</double>
        </property>
        <property name="value">
         <double>0.000000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="CustomPushButton" name="aspectLockButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="toolTip">
      <string>Keep Aspect Ratio</string>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton:hover
{
	background-color:#cdcdcd;
	border-radius:3px;
}

QPushButton:checked
{
	background-color:#c0c0c0;
	border-radius:3px;
}
</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../../../rc.qrc">
       <normaloff>:/uiIcon/src/ui/resource/icon/ic_lock_open_black_18dp.png</normaloff>
       <normalon>:/uiIcon/src/ui/resource/icon/ic_lock_black_18dp.png</normalon>:/uiIcon/src/ui/resource/icon/ic_lock_open_black_18dp.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>18</width>
       <height>18</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame_4">
     <property name="minimumSize">
      <size>
       <width>106</width>
       <height>0</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>4</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>12</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="height">
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Yu Gothic</family>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">#height
{
color: rgb(102, 102, 102);
}
#height:disabled
{
color: rgb(140, 140, 140);
}</string>
        </property>
        <property name="text">
         <string>height: </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="heightEdit">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Yu Gothic</family>
          <pointsize>10</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="focusPolicy">
         <enum>Qt::StrongFocus</enum>
        </property>
        <property name="toolTip">
         <string>Image Height</string>
        </property>
        <property name="styleSheet">
         <string notr="true">QDoubleSpinBox
{
background-color: transparent;
border: 1px solid transparent;
border-bottom-color:#ccc
}
QDoubleSpinBox::up-arrow
{
image: url(:/uiIcon/src/ui/resource/icon/up_arrow.png);
width:7px; height:7px;

}
QDoubleSpinBox::down-arrow
{
image: url(:/uiIcon/src/ui/resource/icon/down_arrow.png);
}
QDoubleSpinBox::down-arrow:disabled
{
image: url(:/uiIcon/src/ui/resource/icon/down_arrow_disabled.png);
}
QDoubleSpinBox::up-arrow:disabled
{
image: url(:/uiIcon/src/ui/resource/icon/up_arrow_disabled.png);
}</string>
        </property>
        <property name="wrapping">
         <bool>false</bool>
        </property>
        <property name="frame">
         <bool>false</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="keyboardTracking">
         <bool>true</bool>
        </property>
        <property name="prefix">
         <string/>
        </property>
        <property name="minimum">
         <double>0.000000000000000</double>
        </property>
        <property name="maximum">
         <double>1000.000000000000000</double>
        </property>
        <property name="value">
         <double>0.000000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame_5">
     <property name="minimumSize">
      <size>
       <width>116</width>
       <height>0</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>12</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>20</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="rotate">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>17</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Yu Gothic</family>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">#rotate
{
color: rgb(102, 102, 102);
}
#rotate:disabled
{
color: rgb(140, 140, 140);
}</string>
        </property>
        <property name="text">
         <string>rotate: </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="rotateEdit">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Yu Gothic</family>
          <pointsize>10</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="focusPolicy">
         <enum>Qt::ClickFocus</enum>
        </property>
        <property name="toolTip">
         <string>Image Rotation</string>
        </property>
        <property name="styleSheet">
         <string notr="true">QSpinBox
{
background-color: transparent;
border: 1px solid transparent;
border-bottom-color:#ccc
}
QSpinBox::up-arrow
{
image: url(:/uiIcon/src/ui/resource/icon/up_arrow.png);
width:7px; height:7px;

}
QSpinBox::down-arrow
{
image: url(:/uiIcon/src/ui/resource/icon/down_arrow.png);
}
QSpinBox::down-arrow:disabled
{
image: url(:/uiIcon/src/ui/resource/icon/down_arrow_disabled.png);
}
QSpinBox::up-arrow:disabled
{
image: url(:/uiIcon/src/ui/resource/icon/up_arrow_disabled.png);
}</string>
        </property>
        <property name="wrapping">
         <bool>false</bool>
        </property>
        <property name="frame">
         <bool>false</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="suffix">
         <string/>
        </property>
        <property name="prefix">
         <string/>
        </property>
        <property name="minimum">
         <number>-180</number>
        </property>
        <property name="maximum">
         <number>180</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="CustomPushButton" name="actualFitButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>36</width>
       <height>36</height>
      </size>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="toolTip">
      <string>Image Actual Size</string>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton:hover
{
	background-color:#cdcdcd;
	border-radius:3px;
}

QPushButton:pressed
{
	background-color:#c0c0c0;
	border-radius:3px;
}
</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../../../rc.qrc">
       <normaloff>:/uiIcon/src/ui/resource/icon/ic_photo_size_select_large_black_24px.svg</normaloff>:/uiIcon/src/ui/resource/icon/ic_photo_size_select_large_black_24px.svg</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CustomPushButton</class>
   <extends>QPushButton</extends>
   <header>ui/widgets/customPushButton.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../rc.qrc"/>
 </resources>
 <connections/>
</ui>
